colossalai/graph/tests/test_parallel_1d.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+
+from functools import partial
```

Comment:
This file should be put into the `tests` folder, otherwise it will not be checked by CI. The graph modules can be moved to the `colossalai.fx` module as well.